docs/conf/eu/2017/news/announcing-events-volunteers.rst

```diff
@@ -0,0 +1,62 @@
+:template: 2017/eu-content.html
+
+.. post:: July 19, 2017
+   :tags: 2017, prague, events, volunteers
+
+Announcing Events and Call for Volunteers
+=========================================
+
+The conference is now just under two months away, and we hope you’re getting geared up for an exciting Write the Docs experience in beautiful Prague, Czech Republic.
+
+`Tickets <http://www.writethedocs.org/conf/eu/2017/tickets>`_ are selling quickly – thanks to everyone who has already purchased theirs! We do expect the conference to sell out fully again this year, so hop off that fence you might be sitting on!
+
+If you're still debating whether to attend, check out the `Sessions <http://www.writethedocs.org/conf/eu/2017/speakers/>`_ page for the full list of presentations.
+
+Events and Activities
+---------------------
+
+As always, Write the Docs events are more than just talks. Every year we try to maximize our attendees experience and invest in the social experience of being a part of our community.
```

Comment:
That still reads a bit passive voice. Needs less abstraction I think. "Every year we make sure to hold lots of social events to allow people to connect and form a community", or something.